public/app/features/dashboard-scene/settings/VariablesEditView.tsx

```diff
@@ -1,30 +1,137 @@
 import React from 'react';
 
 import { PageLayoutType } from '@grafana/data';
-import { SceneComponentProps, SceneObjectBase } from '@grafana/scenes';
+import { SceneComponentProps, SceneObjectBase, SceneVariables, sceneGraph } from '@grafana/scenes';
 import { Page } from 'app/core/components/Page/Page';
 
+import { DashboardScene } from '../scene/DashboardScene';
 import { NavToolbarActions } from '../scene/NavToolbarActions';
 import { getDashboardSceneFor } from '../utils/utils';
 
 import { DashboardEditView, DashboardEditViewState, useDashboardEditPageNav } from './utils';
-
+import { VariableEditorList } from './variables/VariableEditorList';
 export interface VariablesEditViewState extends DashboardEditViewState {}
 
 export class VariablesEditView extends SceneObjectBase<VariablesEditViewState> implements DashboardEditView {
+  public static Component = VariableEditorSettingsListView;
+
   public getUrlKey(): string {
     return 'variables';
   }
 
-  static Component = ({ model }: SceneComponentProps<VariablesEditView>) => {
-    const dashboard = getDashboardSceneFor(model);
-    const { navModel, pageNav } = useDashboardEditPageNav(dashboard, model.getUrlKey());
+  private get _dashboard(): DashboardScene {
+    return getDashboardSceneFor(this);
+  }
+
+  public getDashboard(): DashboardScene {
+    return this._dashboard;
+  }
```

Comment:
The private `_dashboard` getters seems unnecessary, can't see what it brings, seems to be used only in the public method?

Reply:
I removed it ✔️